core/src/it/scala/org/ensime/intg/BasicWorkflow.scala

```diff
@@ -86,7 +86,23 @@ class BasicWorkflow extends EnsimeSpec
           //-----------------------------------------------------------------------------------------------
           // symbolAtPoint
           project ! SymbolAtPointReq(Left(fooFile), 128)
-          val symbolAtPointOpt: SymbolInfo = expectMsgType[SymbolInfo]
+          expectMsgType[SymbolInfo]
+
+          project ! SymbolAtPointReq(Left(fooFile), 68) // scala.Int symbol
+          val sourcePosition = expectMsgType[SymbolInfo].`type`.pos.get
+          val sourceFile = sourcePosition match {
+            case OffsetSourcePosition(file, _) =>
+              file
+            case _ =>
+              EnsimeFile("empty")
+          }
+
+          project ! TypecheckFilesReq(List(Right(SourceFileInfo(sourceFile, None, None))))
+          expectMsg(VoidResponse)
+          asyncHelper.expectMsg(FullTypeCheckCompleteEvent)
+
+          project ! SymbolAtPointReq(Right(SourceFileInfo(sourceFile, None, None)), 720)
```

Comment:
What do you mean by asking for semantic information? 
I just added `case _ => EnsimeFile("empty")` to get rid of the warning. 
I kind of copied `GoToDefinition`'s idea. Getting location of source file from the `TypeInfo`.